doc/releases/squid.rst

```diff
@@ -0,0 +1,561 @@
+  trim the log every second (`mds_log_trim_upkeep_interval` config). Also, a
+  couple of configs govern how much time the MDS spends in trimming its logs.
+  These configs are `mds_log_trim_threshold` and `mds_log_trim_decay_rate`.
+* CephFS: Full support for subvolumes and subvolume groups is now available
+* CephFS: The `subvolume snapshot clone` command now depends on the config
+  option `snapshot_clone_no_wait` which is used to reject the clone operation
+  when all the cloner threads are busy. This config option is enabled by
+  default which means that if no cloner threads are free, the clone request
+  errors out with EAGAIN.  The value of the config option can be fetched by
+  using: `ceph config get mgr mgr/volumes/snapshot_clone_no_wait` and it can be
+  disabled by using: `ceph config set mgr mgr/volumes/snapshot_clone_no_wait
+  false`
+  for snap_schedule Manager module.
+* CephFS: Commands ``ceph mds fail`` and ``ceph fs fail`` now require a
+  confirmation flag when some MDSs exhibit health warning MDS_TRIM or
+  MDS_CACHE_OVERSIZED. This is to prevent accidental MDS failover causing
+  further delays in recovery.
+* CephFS: fixes to the implementation of the ``root_squash`` mechanism enabled
+  via cephx ``mds`` caps on a client credential require a new client feature
+  bit, ``client_mds_auth_caps``. Clients using credentials with ``root_squash``
+  without this feature will trigger the MDS to raise a HEALTH_ERR on the
+  cluster, MDS_CLIENTS_BROKEN_ROOTSQUASH. See the documentation on this warning
+  and the new feature bit for more information.
+* CephFS: Expanded removexattr support for cephfs virtual extended attributes.
+  Previously one had to use setxattr to restore the default in order to
+  "remove".  You may now properly use removexattr to remove. You can also now
+  remove layout on root inode, which then will restore layout to default
+  layout.
+* CephFS: cephfs-journal-tool is guarded against running on an online file
+  system.  The 'cephfs-journal-tool --rank <fs_name>:<mds_rank> journal reset'
+  and 'cephfs-journal-tool --rank <fs_name>:<mds_rank> journal reset --force'
+  commands require '--yes-i-really-really-mean-it'.
+* CephFS: "ceph fs clone status" command will now print statistics about clone
+  progress in terms of how much data has been cloned (in both percentage as
+  well as bytes) and how many files have been cloned.
+* CephFS: "ceph status" command will now print a progress bar when cloning is
+  ongoing. If clone jobs are more than the cloner threads, it will print one
+  more progress bar that shows total amount of progress made by both ongoing
+  as well as pending clones. Both progress are accompanied by messages that
+  show number of clone jobs in the respective categories and the amount of
+  progress made by each of them.
+* cephfs-shell: The cephfs-shell utility is now packaged for RHEL 9 / CentOS 9
+  as required python dependencies are now available in EPEL9.
```

Comment:
Please add:

https://github.com/ceph/ceph/pull/52196/files#diff-e5807f35e3c85ea0c57bba6c6713ace9a5f379edd4ec033f0ac2ac81ccad9972R98

which should have been in the v19.2.0 notes.

Reply:
ACK